test/settings_test.rb

```diff
@@ -179,7 +179,7 @@ def test_implementations_are_independent
     ::Preferences.create var: 'test',  value: 'preferences foo'
     ::Preferences.create var: 'test2', value: 'preferences bar'
 
-    refute_match ::Settings.all_settings, ::Preferences.all_settings
+    refute_equal ::Settings.all_settings, ::Preferences.all_settings
```

Comment:
This test fails when using `refute_match` due to `HashWithIndifferentAccess` not responding to `=~`, which is what the minitest assertion uses to perform the assertion. Changing to `refute_equal` gets the test passing again.

Failure text:
```
1) Failure:
SettingsTest#test_implementations_are_independent [test/settings_test.rb:182]:
Expected {"test" => "foo", "test2" => "bar"} (ActiveSupport::HashWithIndifferentAccess) to respond to #=~.
```